### ISSUE TITLE
Overview Page UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#30]
## User story
1. As a user
2. I want / need to see an overview page after logging in
3. So that I know I successfully logged in and can now book desks

## Acceptance criteria
* The page should be the first page seen after logging in
* The page should show the text: "Hello username, welcome back to Deskstar"
* The page should contain a button that runs the logout function

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
